### PR TITLE
CDAP-4973 Logging documentation revisions

### PR DESCRIPTION
**Pull request by awholegunch**
Fails [Quick Build 3](http://builds.cask.co/browse/CDAP-DOB132-3), though unrelated to this PR.

Fix for https://issues.cask.co/browse/CDAP-4973 (Describe logging location).

Adds the location of logs for Standalone and Distributed CDAP.
Consolidates the logging information into a single file (from three).
Cleans up numerous documentation nits.
Cleans up the CDAP UI page (removes out-dated info).

- [Logging and Monitoring Page](http://builds.cask.co/artifact/CDAP-DOB132/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/admin-manual/operations/logging.html)
- [Operations Index](http://builds.cask.co/artifact/CDAP-DOB132/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/admin-manual/operations/index.html)
- [Admin Manual Index](http://builds.cask.co/artifact/CDAP-DOB132/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/admin-manual/index.html)

### CODE REVIEW

File: cdap-docs/admin-manual/source/index.rst [hunk 1]
```diff
@@ -96,26 +96,23 @@ installation and its security configuration.
 
 - |operations|_
 
-    .. |logging| replace:: **Logging:**
+    .. |logging| replace:: **Logging and Monitoring:**
     .. _logging: operations/logging.html
 
-    - |logging|_ Covers **CDAP support for logging** through standard SLF4J (Simple Logging Facade for Java) APIs.
+    - |logging|_ CDAP collects **logs and metrics** for all of its internal services, while CDAP itself
```
Comment (sreevatsanraman) [inline cdap-docs/admin-manual/source/index.rst]: It is not only for internal services. But also for applications. We do collect application logs and user metrics.

File: cdap-docs/admin-manual/source/operations/logging.rst [hunk 2]
```diff
@@ -1,16 +1,44 @@
...
-============================================
+.. _logging-monitoring:
+
+======================
+Logging and Monitoring
+======================
+
+Logs and Metrics
+================
+CDAP collects logs and metrics for all of its internal services. Being able to view these
```
Comment (sreevatsanraman) [inline cdap-docs/admin-manual/source/operations/logging.rst]: Not only for internal services. But also for applications.

**Comment by awholegunch** [comment]
The last commit (which I reversed) was an accident to the wrong branch.

File: cdap-docs/admin-manual/source/index.rst [hunk 3]
```diff
@@ -96,26 +96,23 @@ installation and its security configuration.
 
 - |operations|_
 
-    .. |logging| replace:: **Logging:**
+    .. |logging| replace:: **Logging and Monitoring:**
     .. _logging: operations/logging.html
 
-    - |logging|_ Covers **CDAP support for logging** through standard SLF4J (Simple Logging Facade for Java) APIs.
+    - |logging|_ CDAP collects **logs and metrics** for all of its internal services and user applications,
+      while CDAP itself can be **monitored through external systems**. Covers **log location**, **logging
```
Comment (sreevatsanraman) [inline cdap-docs/admin-manual/source/index.rst]: This sentence appears to be complete.

File: cdap-docs/admin-manual/source/operations/index.rst [hunk 4]
```diff
@@ -13,20 +13,22 @@ Operations
...
+    CDAP UI <cdap-ui>
+
 
-.. |logging| replace:: **Logging:**
+.. |logging| replace:: **Logging and Monitoring:**
 .. _logging: logging.html
 
-- |logging|_ Covers **CDAP support for logging** through standard SLF4J (Simple Logging Facade for Java) APIs.
+- |logging|_ CDAP collects **logs and metrics** for all of its internal services, while CDAP itself
+  can be **monitored through external systems**. Covers **log location**, **logging
```
Comment (sreevatsanraman) [inline cdap-docs/admin-manual/source/operations/index.rst]: same here.

File: cdap-docs/admin-manual/source/index.rst (hunk 3 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/index.rst]: Are you suggesting just have the first sentence, and drop the second? (See the links above for rendered versions of these pages.)

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**